src/controllers/video-test.controller.js

```diff
@@ -1,20 +1,21 @@
 import { cmcdExtractorService } from '../services/cmcd-extractor.service.js';
 import { VIDEO_TEST_URL } from '../config.js';
-import jsLogger from 'js-logger';
+import log from '../utils/logger.js';
 
 export const videoTest = async (req, res) => {
-
+    const id = req.params['id'];
+    
     try {
         const dateStart = new Date().toISOString();
-        const { filename } = req.params
+        const { filename } = req.params;
         const reqURI = VIDEO_TEST_URL.concat(filename);
-        const {headers, data} = await cmcdExtractorService(req, reqURI, {}, dateStart);
-        res.header(headers)
+        const {headers, data} = await cmcdExtractorService(id, req, reqURI, {}, dateStart);
+        res.header(headers);
         data.pipe(res);
     } catch (error) {
         if (error.response) {
             // The client was given an error response (5xx, 4xx)
-            jsLogger.error(error.response.status, error.response.statusText);
+            log(id, {'error.response.status': error.response.status, 'error.response.statusText': error.response.statusText}, error);

```

Comment:
Change the error.response.status for status and error.respons.statusText for statusText